documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc

```diff
@@ -16,16 +22,16 @@ However, we offer a CLI command to mark the current topology stable for a specif
 topology set-stable cacheName
 ----
 +
-* If the current topology is missing more nodes than the number of owners, the force flag is necessary to confirm the operation.
+* If the number of nodes missing from the current topology is more than or equal to the number of owners, the force flag is necessary to confirm the operation.
 +
 [source,options="nowrap",subs=attributes+]
 ----
 topology set-stable cacheName -f
 ----
-
-For more information about the [command]`topology set-stable` command, run [command]`topology set-stable -h`.
-
++
 [IMPORTANT]
 ====
-Manually installing a topology can lead to data loss, only perform this operation if the initial topology cannot be recreated.
+If you use the force flag, you might lose cache entries. If all the caches do not have the same number of owners, `numOwners`, then only some caches are affected. You can examine the only the caches with a different number of owners.
```

Comment:
It seems to mix a couple of concepts. I agree with the force flag part. I didn't get the second part, though. Can you elaborate more?

Also, one thing worth adding is the stating the command applies to a single command. The user would need to execute the command for each cache they want to set the topology.

Reply:
Thanks, @jabolina, updated, wdyt?